cluster/core/src/main/resources/reference.conf

```diff
@@ -2,21 +2,25 @@
 lagom.cluster {
 
   # The cluster node will join itself if akka.cluster.seed-nodes is not configured.
-  # In dev-mode this setting will be on, otherwise the default is off. It's possible 
+  # In dev-mode this setting will be on, otherwise the default is off. It's possible
   # to override that by defining akka.cluster.seed-nodes or set this property to off in
   # the application.conf
   join-self = ${lagom.defaults.cluster.join-self}
-  
+
   # The ActorSystem is terminated when the cluster member
   # is removed. The delay is to give ClusterSingleton actors
   # some time to stop gracefully.
   terminate-system-after-member-removed = 10s
-  
+
   # Exit the JVM forcefully when the ActorSystem has been terminated.
   # This is by default off, but you may want to turn it on in production
   # to restart the process.
+  # The recommended value (depending on the environment) is:
+  #    * development mode : off
+  #    * running tests : off
+  #    * production / staging  : on
```

Comment:
I wonder if we should force it off for dev/test so that people can just safely add it to `application.conf` rather than having to override it in production.

Reply:
👍 I'd work on it in a separate PR?

Reply:
Yeah, agree.

Reply:
I added a [comment](https://github.com/lagom/lagom/issues/975#issuecomment-327999135) to #975 to check this. It's probably not worth addressing for 1.3.x.